app/queue.py

```diff
@@ -112,6 +121,11 @@ def __move_to_inflight(self, in_flight_key: str, count: int) -> list[str]:
         decoded = [result.decode("utf-8") for result in results]
         return decoded
 
+    def move_from_inflight(self, in_flight_key: str):
+        self.scripts[self.LUA_MOVE_FROM_INFLIGHT](args=[in_flight_key, self._inbox])
```

Comment:
Ideally, the new lua script would just take the expiration period and the inflight pattern to match to move the notifications back into the inbox. We can sync if necessary to provide more details on what I am suggesting if you want.